app/services/pii/session_store.rb

```diff
@@ -26,26 +23,10 @@ def load
     # @param [Pii::Attributes] pii
     def put(pii, expiration = 5.minutes)
       session_data = {
```

Comment:
we could probably remove this and have the tests use the other class directly?

Reply:
I think since it includes the JSON formatting in a way that is expected by `load` it makes sense to keep it separate.